src/validations.ts

```diff
@@ -0,0 +1,156 @@
+import { fullNameIfAvailable } from "./util.ts";
+import { AddressBookEntry, SafeTransaction, TxDetails } from "./models.ts";
+import { COW_TOKEN } from "./constants.ts";
+
+export  function notifyIfNot(condition: boolean, msg: string): boolean {
+  if (!condition) {
+    console.log(msg);
+    return false;
+  }
+  return true;
+}
+export function notifyIfDifferent(
+  real: unknown,
+  expected: unknown,
+  msg: string
+): boolean {
+  if (real !== expected) {
+    console.log(`Expected: ${expected}; is: ${real}.`, msg);
+    return false;
+  }
+  return true;
+}
```

Comment:
This doesn't perform the intended purpose of stopping the final output from appearing if something is bad. If we only care about the error log, we can just remove the output value.
Something we could do for later is having this function also take an `errorStack` vector that collects all error messages. Then in the end we check if the vector is empty to determine that no error was encountered. If there is an error, then we can print out all errors nicely in the interface. (Right now, `console.log` would only output to the web console.)

Reply:
Ya, I agree its not quite what was originally intended. Are there any suggestions you have that I could/should make right now or perhaps an issue we could record on the matter?

Reply:
Everything is experimental so I don't think we need issues now! I'd still remove the return value since it's weird, later we'll see how to handle the console.

Reply:
That all fine and dandy, but I will also have to remove the `throw Error` since I don't believe the logs will even appear in the console if it throws.

Reply:
Done!